spec/money_spec.rb

```diff
@@ -280,7 +281,9 @@ def expectation.fractional
       end
 
       it "is a BigDecimal when using infinite_precision", :infinite_precision do
-        money = YAML::load serialized
+        money = YAML::load serialized.sub(
+          'infinite_precision: false', 'infinite_precision: true'
```

Comment:
^ this change is tricking the spec as it won't respect the global setting as it used to do previously, introducing a breaking change

Reply:
This will not respect the global setting anymore, because there's no `#initialize` method called when you load a serialized object. And when you serialize it - it stores the value of an `@infinite_precision` instance variable.

Reply:
@antstorm example title is now more meaningful:
```ruby
it "it respects the object's infinite_precision setting" do
```